### PR TITLE
Port websocket fat

### LINKED ISSUE #5207: WSOC: Port websocket fvt test cases to OpenLiberty
WSOC: Port websocket fvt  test cases to OpenLiberty.

### PR DESCRIPTION
**Pull request by isaacrivriv**
partially fixes #5207

### CODE REVIEW

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#run-libby-bot

**Comment by isaacrivriv** [comment]
#build

**Comment by isaacrivriv** [comment]
#run-libby-bot

**Comment by isaacrivriv** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wsoc_fat

File: dev/io.openliberty.wsoc.internal_fat/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src:\
+    fat/src,\
+    test-applications/basic.jar/src,\
+    test-applications/basic.war/src,\
+	test-applications/websocket11.war/src
+
```
Comment (pnicolucci) [inline dev/io.openliberty.wsoc.internal_fat/bnd.bnd:19]: Can you just clean up this white space difference ?

File: dev/io.openliberty.wsoc.internal_fat/build.gradle [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    // Define G:A:V coordinates of each dependency
+    dependencies {
+      junit 'junit:junit:4.12'
+    }
+    // Copy the dependencies wherever they may be needed
+    copy {
+      // Copy junit jars into a specific folder so they can be referenced
+      from configurations.junit
+      into "${buildDir}/autoFVT/test-applications/basic.war/resources/WEB-INF/lib"
+    }
```
Comment (pnicolucci) [inline dev/io.openliberty.wsoc.internal_fat/build.gradle:41]: Is this necessary? I think we get junit for free for any FAT buckets, or is there something specific here for the websocket tests?

File: dev/io.openliberty.wsoc.internal_fat/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+-buildpath: \
+    com.ibm.ws.junit.extensions;version=latest,\
+    com.ibm.ws.common.encoder;version=latest,\
+    com.ibm.websphere.javaee.ejb.3.1;version=latest,\
+    com.ibm.websphere.javaee.cdi.1.0;version=latest,\
+    com.ibm.websphere.javaee.servlet.3.1;version=latest,\
+    com.ibm.websphere.javaee.el.3.0;version=latest,\
+    com.ibm.websphere.javaee.jsp.2.3;version=latest,\
+    com.ibm.ws.jsp.2.3;version=latest,\
```
Comment (pnicolucci) [inline dev/io.openliberty.wsoc.internal_fat/bnd.bnd:34]: do we actually need jsp/el/servlet here?

File: dev/io.openliberty.wsoc.internal_fat/bnd.bnd (hunk 1 above)
Comment (isaacrivriv) [reply dev/io.openliberty.wsoc.internal_fat/bnd.bnd:19]: Sounds good, Ill make the change

File: dev/io.openliberty.wsoc.internal_fat/build.gradle (hunk 2 above)
Comment (isaacrivriv) [reply dev/io.openliberty.wsoc.internal_fat/build.gradle:41]: Yeah so we do get the junit file for free but inside the war app itself it needs the jar for some assertions it makes and other junit related functions. In CL it was already in the folder but we can't have that in OL so I removed it and add it when building the project and later the app gets build with it. If I just remove it and dont add it the test fail with a class not found exception

File: dev/io.openliberty.wsoc.internal_fat/bnd.bnd (hunk 3 above)
Comment (isaacrivriv) [reply dev/io.openliberty.wsoc.internal_fat/bnd.bnd:34]: I was basing myself on what was included in the classpath of the wsoc test in CL. I tried leaving them out but got some errors about class not found exceptions. Ill try again to see if I run into the same errors

**Comment by isaacrivriv** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wsoc_fat

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**